docs/topics/overriding-rules.adoc

```diff
@@ -29,6 +29,8 @@ Rules from the original ruleset that are not overridden by the new ruleset are r
 
 . Ensure that you keep the same rule and ruleset IDs. When you copy the original rule XML, this ensures that the IDs match.
 
+. Ensure that `targetTechnology` within the override ruleset matches one of the targets that you specified for running the analysis.
+
```

Comment:
Suggestion: "...that the `targetTechnology` included in the override ruleset matches..." .

Reply:
@RichardHoch 
Yes, i would have certainly put 'the' there, had `targetTechnology` been a common noun. But the backticks make it sort of a proper noun - won't you agree? 
But how about I change it to this: "Ensure that the target technology in the override ruleset matches..."?

Reply:
Your solution works. I don't think the backticks make the item a proper noun, but some proper nouns such "the Cabinet" take "the" anyway. So either go with "the `targetTechnology` .." or "the target technology," whichever you think is best.